### PR TITLE
Fix: wrong info displayed when quickly navigate from a channel to another

### LINKED ISSUE #1545: When quickly switching between two channels, the name of the second channel will be displayed, while the videos of the first channel will be displayed
**Behavior of the program**
When clicking on a channel you have subscribed to, and then quickly clicking on a second channel you have subscribed to, it will display the name and logo of the second channel. However, instead of the second channel's videos, it will display the first channels videos.

**Expected behavior**
By switching quickly between two channels, the name of the second channel will be display, while the videos of the first channel will be displayed.

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Quickly click on one channel.
2. Then, quickly click on a different channel.
3. See error.

**Screenshots**
Skjermbilde fra 2021-07-25 20-58-52

**Environment Information (please complete the following information):**
 - OS: Linux Mint.
 - OS Version: 20.2.
 - FreeTube version: 13.2 Beta.
 - Installation Method (If applicable): Package manager.
 - Primary API Used: Invidious API.

### PR DESCRIPTION
**Pull request by hockerschwan**
**Pull Request Type**
- [x] Bugfix

**Related issue**
closes #1545

**Description**
Holds id in `getChannelInfoInvidious` etc. and compares to this.id, if they are different (route changed), do nothing.

**Screenshots (if appropriate)**
Before:
<video src="https://user-images.githubusercontent.com/80553357/169692829-4ff6cc7e-bb4a-4d21-9c9c-64141c790d00.mp4">

After:
<video src="https://user-images.githubusercontent.com/80553357/169692835-16c137be-daa9-4c17-b92f-bea410e2ae74.mp4">

**Desktop (please complete the following information):**
 - OS: Windows10
 - OS Version: 21H1
 - FreeTube version: 75d59a54990280c08488cb39ba136d2b38c9d0fe

### CODE REVIEW

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
This also happens on the local api

https://user-images.githubusercontent.com/73130443/169694151-850a226e-eb32-4a88-b788-59ec664fa3f7.mp4

**Comment by hockerschwan** [comment]
Local api added

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — COMMENTED** [review]
LGTM!

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]
LGTM!

File: src/renderer/views/Channel/Channel.js [hunk 1]
```diff
@@ -249,7 +249,12 @@ export default Vue.extend({
 
     getChannelInfoLocal: function () {
       this.apiUsed = 'local'
+      const id = this.id
```
Comment (PikachuEXE) [inline src/renderer/views/Channel/Channel.js:252]: Give it a better name, same for other changes
```suggestion
      const expectedId = this.id
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

File: src/renderer/views/Channel/Channel.js (hunk 1 above)
Comment (hockerschwan) [reply src/renderer/views/Channel/Channel.js]: fixed.

**Comment by hockerschwan** [comment]
yes please
(changed variable names, no functional changes. )

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]
LGTM!

**Review by PikachuEXE — APPROVED** [review]

**Pull request merged**